workflow/hosts/awspw.yaml

```diff
@@ -27,5 +27,4 @@ MAKE_ACFTBUFR: 'NO'
 DO_TRACKER: 'NO'
 DO_GENESIS: 'NO'
 DO_METP: 'NO'
-SUPPORT_WAVES: 'NO'
-SUPPORTED_RESOLUTIONS: ['C48', 'C96'] # TODO: Test and support all cubed-sphere resolutions.
+SUPPORTED_RESOLUTIONS: ['C48', 'C96', 'C384', 'C768'] # TODO: Test and support all cubed-sphere resolutions.
```

Comment:
Is 192 not supported yet?

Reply:
I won't say that, but I have not run any C192 case yet. 
Let me add it there.